CefSharp/Internals/JavascriptObject.cs

```diff
@@ -40,9 +44,40 @@ public class JavascriptObject //: DynamicObject maybe later
         public List<JavascriptProperty> Properties { get; private set; }
 
         /// <summary>
+        /// Indicate if the <see cref="JavascriptObject" /> is null, so that on browser side we don't need to create an cef object.
+        /// </summary>
+        [DataMember]
+        public bool IsNull { get; private set; }
+
+        /// <summary>
+        /// Gets or sets a delegate which is called when binding occurred.  
+        /// </summary>
+        internal Action LateBinding { private get; set; }
+
+        /// <summary>
+        /// Calls <see cref="LateBinding" /> if not already bound.
+        /// </summary>
+        /// <returns>this, so that calls can be chained.</returns>
+        internal JavascriptObject Bind()
+        {
+            if (!bound && LateBinding != null)
+                LateBinding();
+            bound = true;
+            return this;
+        }
+
+        /// <summary>
         /// Gets or sets the value.
         /// </summary>
-        public object Value { get; set; }
+        public object Value
```

Comment:
It won't work because Value is always null on the browser side. It's not transmitted over the wire.